pkg/controller/service/reconciler.go

```diff
@@ -104,6 +106,13 @@ func NewServiceReconciler(consoleClient client.Client, config *rest.Config, refr
 		return nil, err
 	}
 
+	go func() {
+		//nolint:all
+		_ = wait.PollImmediateInfinite(time.Minute*5, func() (done bool, err error) {
+			return false, CapabilitiesAPIVersions(discoveryClient)
```

Comment:
if this returns err does the poller stop?

Reply:
yes, but I've changed it to print the error log